tox.ini

```diff
@@ -72,7 +72,7 @@ commands =
 # E123, E125 skipped as they are invalid PEP-8.
 # see: http://flake8.readthedocs.io/en/latest/config.html#settings
 show-source = True
-ignore = E123,E125
+ignore = E123,E125,E402
```

Comment:
skipping this error because 
it is import not in top error 

anmol used logic here https://github.com/Tendrl/node-agent/blob/master/tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_brick_utilization.py#L12

import between sys so it isvalid case so i am skipping this error